plugins/uefi/fu-plugin-uefi.c

```diff
@@ -439,6 +440,12 @@ fu_plugin_uefi_load_config (FuPlugin *plugin, FuDevice *device)
 	fu_device_set_metadata_boolean (device,
 					"RequireShimForSecureBoot",
 					!disable_shim);
+
+	/* check if using UEFI removeable path */
+	enable_removable_path = fu_plugin_get_config_value_boolean (plugin, "EnableUefiRemovablePath");
+	fu_device_set_metadata_boolean (device,
+					"RequireRemovablePath",
```

Comment:
My proposal was to fallback to this if shim doesn't exist so confined snap could work on other distros than UC.  Is there a reason not to do this?

Reply:
No, I tried to follow up your approach, but seems I misunderstand what you pointed to me when I thought a new option should be loaded here. Well..I'm not very familiar with these codes.

Reply:
Take a look at my two new comments and I think it should be clearer.